docs/v1/user-story

```diff
@@ -0,0 +1,124 @@
+using a service that lets me have my own instance and know that it’s completely isolated from other customers, so when there 
+is an update, I can choose whether or not to deploy it.  I’m now starting to think harder about the benefits of transparency 
+in vendor deployments vs. the benefits of fully managed services.  But I have to admit just how cool it is that I got the 
+message so quickly in such an obvious way.  They made an update and published it to the store.  Kubernetes’ record of the 
+database type in the service catalog that was created when I bought the service noticed the new store entry.  I was notified 
+me about the update immediately since I had an instance of it running and my chat app bound to that instance.  Now to do the 
+DB upgrade... 
+
+12. Upgrade service instances to new service type version
+Wow, the version upgrade was just as easy as the “small” to “large”!   All I needed was one update command and the database 
+was quickly and non-disruptively upgraded.  (OK, I skipped the part where I tested the darn thing for a week - but you get 
+the idea)  That’s just amazing.  Again, kudos to whoever is working that update service broker over at the database vendor!
+
+13. Discover unapproved update to a service’s resources  
+While I was on a week long vacation, the interns got a call about some customer issue and they worked it for a couple days 
+and determined that the container size needed to be changed.  And they just changed it on their own since they did not want 
+to bother me on vacation.  The service definition I made clearly did not make that a parameter they could change, though and 
+they didn’t think to tell me about it when I got back since everything went back to normal by then.  But, I look at my 
+messages and there’s this alert telling me that the service instance that’s handling all our support communications is out 
+of compliance because of this setting.  I was initially upset about this, but when I looked at it in more detail, I 
+discovered that the interns were spot on.  After a certain point of load (which will not stop going up), we needed to start 
+using bigger containers and not just keep scaling out more and more tiny ones.  (Not only did the immediate problem go away, 
+but paradoxically, the cost went down which was nice - they solved another problem I’m sure I would have had next quarter)  
+But, now we need to roll this out more deliberately so that the setting sticks and that new instances are deployed properly 
+as well.  It’s a good thing I got the notification or I’d have never known.  Thanks Kubernetes!
+
+14. Update type, create update template, and redeploy service 
+So, I do for my service what the the database people did for me.  I created a new version, tested it out, and deployed it to 
+the catalog (without any marketplace in the way, since this is an internal app).  Now, I’m getting messages about my chat 
```

Comment:
This is the second time the marketplace was mentioned, but it hasn't been described. Since this is a user story doc, it might be good to indicate exactly why and how you, the app developer, would use it.

Also, it'd probably be good to add to the root README.md document as a term. It seems to be a new concept that I've overheard a few times in passing.